databricks_cli/click_types.py

```diff
@@ -98,12 +98,12 @@ class SecretPrincipalClickType(ParamType):
 
 class PipelineSpecClickType(ParamType):
     name = 'SPEC'
-    help = 'The path to the pipelines deployment spec file'
+    help = 'The path to the pipelines deployment spec file.'
 
 
 class PipelineIdClickType(ParamType):
     name = 'PIPELINE_ID'
-    help = 'Delta Pipeline ID'
+    help = 'The Delta Pipeline ID.'
```

Comment:
Should be "The Delta Live Tables pipeline ID" or just "The pipeline ID."

Reply:
yeah just "The Pipeline ID" should be good here.